### PR TITLE
Feat/vic 226 product walkthrough

### PR DESCRIPTION
**Pull request by adnanalicic**

### CODE REVIEW

File: api/userservice.yaml [hunk 1]
```diff
@@ -836,6 +836,23 @@ paths:
           description: CONFLICT - invalid/unavailable username
         500:
           description: INTERNAL SERVER ERROR - server encountered unexpected condition
+  /users/consultants/toggleWalkThrough:
```
Comment (mebo4b) [inline api/userservice.yaml:839]: Could you please the existing "PATCH" endpoint of /users/data and just add there a new property for the walkthrough?

File: api/userservice.yaml [hunk 2]
```diff
@@ -2074,6 +2091,10 @@ components:
           type: boolean
           example: true
           description: "Is true if feature is enabled. Enables end-to-end encryption in video chat"
+        isWalkThroughEnabled:
+          type: boolean
+          example: true
+          description: "Is true if the walt through is enabled for the user"
```
Comment (mebo4b) [inline api/userservice.yaml:2097]: Typo: walt

File: api/userservice.yaml (hunk 1 above)
Comment (mebo4b) [reply api/userservice.yaml:839]: You can then use all the existing service methods as well. And please add also tests for the new property

**Review by maeventer — CHANGES_REQUESTED** [review]

**Review by maeventer — APPROVED** [review]

File: api/userservice.yaml (hunk 1 above)
Comment (adnanalicic) [reply api/userservice.yaml]: done

File: api/userservice.yaml (hunk 2 above)
Comment (adnanalicic) [reply api/userservice.yaml]: done

**Pull request merged**